[2025-09-19T03:15:00.823+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-19T03:15:00.873+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-19T01:00:00+00:00 [queued]>
[2025-09-19T03:15:00.884+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-19T01:00:00+00:00 [queued]>
[2025-09-19T03:15:00.885+0200] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-19T03:15:00.900+0200] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_customers> on 2025-09-19 01:00:00+00:00
[2025-09-19T03:15:00.910+0200] {standard_task_runner.py:64} INFO - Started process 476 to run task
[2025-09-19T03:15:00.919+0200] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_api_to_bi', 'extract_customers', 'scheduled__2025-09-19T01:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/etl_api_to_bi.py', '--cfg-path', '/tmp/tmpmvc6ntul']
[2025-09-19T03:15:00.924+0200] {standard_task_runner.py:91} INFO - Job 28: Subtask extract_customers
[2025-09-19T03:15:01.054+0200] {task_command.py:426} INFO - Running <TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-19T01:00:00+00:00 [running]> on host 286d1f3cd4b3
[2025-09-19T03:15:01.517+0200] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data' AIRFLOW_CTX_DAG_ID='etl_api_to_bi' AIRFLOW_CTX_TASK_ID='extract_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-09-19T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-19T01:00:00+00:00'
[2025-09-19T03:15:01.518+0200] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-19T03:15:01.583+0200] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-09-19T03:15:01.585+0200] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2025-09-19T03:15:01.634+0200] {python.py:237} INFO - Done. Returned value was: 1
[2025-09-19T03:15:01.635+0200] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-19T03:15:01.694+0200] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_api_to_bi, task_id=extract_customers, run_id=scheduled__2025-09-19T01:00:00+00:00, execution_date=20250919T010000, start_date=20250919T011500, end_date=20250919T011501
[2025-09-19T03:15:01.736+0200] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-19T03:15:01.774+0200] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-19T03:15:01.778+0200] {local_task_job_runner.py:222} INFO - ::endgroup::
